tests/serial/map-if-necessary/three-solvers/helper.hpp

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <vector>
+
+#include "precice/impl/ParticipantImpl.hpp"
+#include "testing/Testing.hpp"
+
+// Coupling is A - C - B, which can be MultiCoupling or Compositional. A-C may be explicitly coupled.
+// A and B always go first.
+// A never sends substeps, B always sends substeps, which should be reflected in the read mappings of C.
```

Comment:
"B always sends substeps" is not correct for the "multi" case, for example, or did I get this wrong?

Reply:
![image](https://github.com/precice/precice/assets/13552216/f5f67762-bd22-4586-9d70-be471524a62c)

Correct